app.test.js

```diff
@@ -0,0 +1,70 @@
+const app = require("./app");
+const supertest = require("supertest");
+const request = supertest(app);
+
+describe("Testing endpoints", () => {
```

Comment:
You have written only one test... Why? 🤔
Also, I could not find where you handle environments change and switch DB's